apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -65,16 +65,18 @@
   "udapp.tooltipText3": "Click to open a bridge for converting L1 mainnet ETH to the selected network currency.",
 
   "udapp._comment_instanceContainerUI.tsx": "libs/remix-ui/run-tab/src/lib/components/instanceContainerUI.tsx",
-  "udapp.deployedContracts": "Deployed Contracts",
+  "udapp.deployedContracts": "Deployed/Unpinned Contracts",
   "udapp.deployAndRunClearInstances": "Clear instances list and reset recorder",
-  "udapp.deployAndRunNoInstanceText": "Currently you have no deployed contracts to interact with.",
-  "udapp.tooltipText6": "Autogenerated generic user interfaces for interaction with deployed contracts",
+  "udapp.deployAndRunNoInstanceText": "Currently you have no unpinned contracts to interact with.",
```

Comment:
I think removing "unpinned" here will better simplify the message. "Currently you have no contracts to interact with."